client/src/components/App.jsx

```diff
@@ -1,119 +1,115 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import axios from "axios";
-import Features from "./Features.jsx";
+import Wifi from "./Wifi.jsx";
+import Modal from "./Modal.jsx";
+import Coffee from "./Coffee.jsx";
 import Icons from "./Icons.jsx";
 import Carousel from "./Carousel.jsx";
 
 class App extends React.Component {
 	constructor(props) {
 		super(props);
 		this.state = {
-			carousel: false
+			carousel: false,
+			wifi: false,
+			coffee: false,
+			name: "",
+			location: "",
+			city: "",
+			country: ""
 		};
-		this.getPhotos = this.getPhotos.bind(this);
+		this.getHostelInfo = this.getHostelInfo.bind(this);
+		this.getLocationInfo = this.getLocationInfo.bind(this);
+		this.toggleCarousel = this.toggleCarousel.bind(this);
 	}
 
-	getPhotos() {
-		// axios
-		// 	.get("/locations/hostels/hosetelId/pictures")
-		// 	.then(response => {
-		// 		console.log(response.data[0]);
-		// 	})
-		// 	.catch(error => {
-		// 		console.log(error);
-		// 	});
+	componentDidMount() {
+		this.getHostelInfo();
```

Comment:
i liked how you modulized ajax requests from componentDidMount

Reply:
thank you! I tried hard to see where I could modularize my code. I even made a wifi, coffee, and modal component.

Reply:
No I'll talk to you more about proptypes. I don't know what that is